src/linagora.esn.unifiedinbox/app/components/composer/body-editor/html/composer-body-editor-html.controller.js

```diff
@@ -60,6 +61,23 @@ angular.module('linagora.esn.unifiedinbox')
       self.onBodyUpdate({ $body: $element.find('.summernote').summernote('code') });
     }
 
+    function onSummernotePaste(event) {
+      event.preventDefault();
+      var text = event.originalEvent.clipboardData.getData('text/plain'),
+        html = event.originalEvent.clipboardData.getData('text/html');
+      const badAttributes = ['line-height'];
+
+      if (html) {
+        var attributeStripper = new RegExp('(' + badAttributes.join('|') + '\s*:\s*(\w+|\d\.\d+)%?\;?)'); // eslint-disable-line
```

Comment:
Try it on https://regex101.com/ for example, there is a problem with you "joined group" priority. `(first-attribute)|(second-attribute)|(last-attribute\s*:\s*(\w+|\d\.\d+)%?\;?)` that is how is is interpreted, I think it is not what you wanted?

Can you explain how does this regex work?

Reply:
https://regex101.com/r/cKAdRC/1/